### ISSUE TITLE
Usergroup (layout.xml) url="[none]" doesn't work (Origin: bugzilla #750666)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5801]
status NEEDINFO severity *normal* in component *general* for *---*
Reported in version *1.8.9.1* on platform *Other*
Assigned to: Dimitri van Heesch

On 2015-06-09 20:24:04 +0000, Alex Turbov wrote:

    â¦ it generates a link like this:
    
    <a href="[none]"><span>Sample</span></a>

On 2015-06-10 07:13:15 +0000, albert wrote:

    What causes a link like you mentioned?
    
    Please attach an example (source + config file in a zip or tar) for which
    you verified it triggers the problem?

On 2016-02-04 10:17:51 +0000, John Olsson wrote:

    (In reply to albert from comment # 1)
    > What causes a link like you mentioned?
    > 
    > Please attach an example (source + config file in a zip or tar) for which
    > you verified it triggers the problem?
    
    A default DoxygenLayout.xml generated with 1.8.11 and this modification at
    the top
    
    <doxygenlayout version="1.0">
      
      
      <navindex>
        <tab type="mainpage" visible="yes" title=""/>
        <tab type="usergroup" title="Subunits" url="[none]">
          <tab type="user" url="@ref conf" title="Conf"/>
          <tab type="user" url="@ref coord" title="Coord"/>
          <tab type="user" url="@ref sync" title="Sync"/>
          <tab type="user" url="@ref common" title="Common"/>
        </tab>
    ...
    
    reproduces the problem. It gives the following error message in Firefox when
    I click on the tab "Subunits"
    
    
    File not found
    
    Firefox can't find the file at /path/to/doc/html/[none].
    
    
    (I have replaced the absolute path reported by Firefox with "/parth/to/")

**Comment by albert-github** [comment]
With the given snippet I can reproduce the problem. The problem here is the:
```
<tab type="usergroup" title="Subunits" url="[none]">
```
and especially the `<tab type="usergroup" title="Subunits" url="[none]">`, `[none]` is in this case not a valid url.
One has either use:
```
<tab type="usergroup" title="Subunits">
```
or
```
<tab type="usergroup" title="Subunits" url="">
```

The problem looks like to be usage, though in the documentation this possibility is explicitly mentions, so it is a bug.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8923

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.3.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
